Source/WebCore/html/FormListedElement.cpp

```diff
@@ -123,9 +124,9 @@ void FormListedElement::formOwnerRemovedFromTree(const Node& formRoot)
 {
     ASSERT(form());
     // Can't use RefPtr here beacuse this function might be called inside ~ShadowRoot via addChildNodesToDeletionQueue. See webkit.org/b/189493.
-    Node* rootNode = &asHTMLElement();
-    auto* currentForm = form();
-    for (auto* ancestor = asHTMLElement().parentNode(); ancestor; ancestor = ancestor->parentNode()) {
+    SUPPRESS_UNCOUNTED_LOCAL Node* rootNode = &asHTMLElement();
```

Comment:
cc @rniwa . I am not convinced we want to suppress here. Odds are we need to make this code safe.